B.py

```diff
@@ -0,0 +1,2 @@
+def hallo():
```

Comment:
```suggestion
def hello():
```
holloではなく、helloではないでしょうか！